pkg/config/testdata/piped/piped-config.yaml

```
@@ -122,6 +122,19 @@ spec:
         slack:
           oauthToken: token
           channelID: testid
+      - name: hookurl-with-mentioned-accounts
+        slack:
+          hookURL: https://slack.com/dev
+          mentionedAccounts:
+            - 'user1'
+            - 'user2'
```

Cover different logic. (This case fails because the trim is not evaluated.)
```suggestion
            - '@user2'
```